core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceFactory.java

```diff
@@ -242,6 +242,12 @@ private static HttpStatus defaultResponseStatus(Method method) {
                                                 List<ResponseConverterFunction> baseResponseConverters,
                                                 List<ExceptionHandlerFunction> baseExceptionHandlers) {
 
+        if (KotlinUtil.getCallKotlinSuspendingMethod() == null && KotlinUtil.maybeSuspendingFunction(method)) {
+            throw new IllegalArgumentException(
+                    "Kotlin suspending functions cannot be used. You should add the armeria-kotlin dependency."
```

Comment:
Did we document about this before? If so, how about adding a URL to the exception message? e.g.

    Kotlin suspending functions are supported only when you added 'armeria-kotlin' as a dependency.
    See https://armeria.dev/.... for more information.

Reply:
OK, I added the URL to the exception message.